test/unit/destructuring.spec.ts

```diff
@@ -51,16 +51,11 @@ test.each(testCases)("in variable declaration (%p)", ({ binding, value }) => {
     `.expectToMatchJsResult();
 });
 
-// TODO: https://github.com/TypeScriptToLua/TypeScriptToLua/issues/695
-// TODO: https://github.com/microsoft/TypeScript/issues/32656
-test.each(testCases.filter(x => x.binding !== "[x, , y]" && x.binding !== "{ x, ...rest }"))(
-    "in exported variable declaration (%p)",
-    ({ binding, value }) => {
-        util.testModule`
+test.each(testCases)("in exported variable declaration (%p)", ({ binding, value }) => {
+    util.testModule`
             export const ${binding} = ${value};
```

Comment:
```suggestion
        export const ${binding} = ${value};
```
Hope one day we'll have multiline suggestions